test-data/unit/check-flags.test

```diff
@@ -1439,7 +1441,7 @@ def f(c: A) -> None:  # E: Missing type parameters for generic type "A"
     pass
 [file mypy.ini]
 \[mypy]
-strict = True
+disallow_any_generics = True
```

Comment:
why replace `strict` with this specific flag

Reply:
```
[case testStrictInConfigAnyGeneric]
# flags: --config-file tmp/mypy.ini
from typing import TypeVar, Generic
```
I'll add a comment

Reply:
`-- maybe a little useless because theres no strict but w/e`